adding apps bulk assign to project guide in to the assessment project (#154)
tidalmigrations/tidalmigrations.github.io: pages/user guides/assessment_projects.md

It's OK to skip the fields you are unsure about. You can always come back to this page and adjust the details as things evolve.

## Bulk assign applications to a project
You also have the ability to bulk assign the applications you selected to a project. Simply navigate to the `Applications` tab under `ASSESS` menu or manually visit `https://<your-subdomain>.tidalmg.com/#/apps`, select the applications you want to assign by click the checkboxes and then click the `Project Assign` button.

{% include image.html file="projects_10.png" %}

> This will brings up the a modal which contains a list of projects and you can search the name of the project by filtering the project name in the search bar above the list.

Reviewer: ```suggestion
This will brings up a modal which contains a list of projects. You can search this list by name of the project by filtering the project name in the search bar above the list.
```

> This will brings up a modal which contains a list of projects. You can search this list by name of the project by filtering the project name in the search bar above the list.

Reviewer: ```suggestion
This will brings up a modal that contains a list of projects. You can search this list by the name of the project by filtering the project name in the search bar above the list.
```

{% include image.html file="projects_11.png" %}

> Additionally, you can easily create a brand new project by simply typing the project name in the search bar and clicking `Create New Project: [Your project name]`. A new project with the name you entered will be created within seconds and you can edit project details later in the project detail view.

Reviewer: ```suggestion
Additionally, you can easily create a brand new project by simply typing the project name in the search bar and clicking `Create New Project: [Your project name]`. Within seconds this will create a new project with the name you entered will be, and you can edit project details later in the project detail view.
```

{% include image.html file="projects_12.png" %}

> Once you have selected the project you want to assign, continue the process by clicking the `Assign N App(s) to Project` button. Depending on the applications you selected, if there is no application that is already being assigned to the project in the selected applications, you will see the confirmation modal directly.

Reviewer: ```suggestion
Once you have selected the project you want to assign, continue the process by clicking the `Assign N App(s) to Project` button. If none of your selected applications have projects assigned to them, you will see the confirmation modal directly.
```

{% include image.html file="projects_13.png" %}

> If there is any selected application which already being assigned to a project, then you will see a warning modal which contains a list of conflicted applications. You can resolve the conflicts by either clicking `Remove` button beside each list item to deselect the application, or clicking `Overwrite` button to force update those conflicted applications to the new project you just selected.

Reviewer: ```suggestion
If there is any selected application which is already assigned to a project, then you will see a warning modal which contains a list of conflicting applications. You can resolve the conflicts by clicking the `Remove` button beside each list item to deselect the application. You can also force the conflicting applications to update to the new project by clicking the `Overwrite` button.```

> If there is any selected application which is already assigned to a project, then you will see a warning modal which contains a list of conflicting applications. You can resolve the conflicts by clicking the `Remove` button beside each list item to deselect the application. You can also force the conflicting applications to update to the new project by clicking the `Overwrite` button.

Reviewer: ```suggestion
If there are any applications with assigned projects, you will see a warning modal containing a list of conflicting applications. You can resolve the conflicts by clicking the `Remove` button beside each list item to deselect the application. You can also force the conflicting applications to update to the new project by clicking the `Overwrite` button.
```

{% include image.html file="projects_14.png" %}

> And finally, you will see the confirmation modal which contains the number of applications you assigned, a clickable link to the selected project so you can go check the updates you just made, and a `Return` button which will close the modal and stay you in the Application page.

Reviewer: ```suggestion
Finally, you will see the confirmation modal which tells you how many applications have been added to the project. It also has a clickable link to the selected project so you can verify the updates you just made. Or you can click the `Return` button which will close the modal and stay you in the Application page.
```

> Finally, you will see the confirmation modal which tells you how many applications have been added to the project. It also has a clickable link to the selected project so you can verify the updates you just made. Or you can click the `Return` button which will close the modal and stay you in the Application page.

Reviewer: ```suggestion
Finally, you will see the confirmation modal, which tells you how many applications have been added to the project. It also has a clickable link to the selected project so you can verify the updates you just made. Or you can click the `Return` button, which will close the modal and keep you on the Application page.
```

Reviewer 2: ```suggestion
Finally, you will see the confirmation modal, which tells you how many applications have been added to the project. It also has a clickable link to the selected project so you can verify the updates you just made. Or you can click the `Return` button, which will close the modal and keep you on the Applications page.
```
